lib/pages/playlist.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+
+class PlayListPage extends StatefulWidget {
+  const PlayListPage({super.key});
+
+  @override
+  State<PlayListPage> createState() => _PlayListPageState();
+}
+
+class _PlayListPageState extends State<PlayListPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Padding(
+        padding: const EdgeInsets.all(8),
+        child: Column(
+          children: [
+            Image.network(
```

Comment:
![image](https://github.com/IEEE-VIT/playtify/assets/84137299/c83ac7d6-0a7d-4fe8-b60f-b89649841556)

Problem of image overflow
there are many ways to fix it, learn how to fix this
also, see what the point of adding an image there I don't get it.